giskard/utils/__init__.py

```diff
@@ -21,19 +23,23 @@ class WorkerPool:
 
     def __init__(self):
         self.pool = None
+        self.nb_cancellable = 0
+        self.max_workers = 0
 
-    def start(self, *args, **kwargs):
+    def start(self, max_workers: int = None):
```

Comment:
I forgot the codestyle stuff, should it be something like `max_workers: int=None`?

Reply:
I did the formatting with black, so should be fine as is